scripts/kconfig/checkconfig.py

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env python3
 
+# NOTE: This script has not been updated for Kconfiglib 2 and will not run
```

Comment:
so why do we keep this script here at all?

Reply:
Thought it might be more likely that someone might update it if it's kept. I have no strong opinions here though.

Reply:
@nashif any comments on this script?